sbws/lib/v3bwfile.py

```diff
@@ -432,6 +435,42 @@ def add_stats(self, **kwargs):
         [setattr(self, k, str(v)) for k, v in kwargs.items()
          if k in STATS_KEYVALUES]
 
+    def add_relays_excluded_counters(self, exclusion_dict):
+        log.debug("Adding relays excluded counters.")
+        for k, v in exclusion_dict.items():
+            setattr(self, k, str(v))
+
+    def add_time_report_all_network(self):
```

Comment:
Hm, function name seems misleading. Isn't this supposed to report time for half the network? (Which is also a bit confusing. Perhaps there should be some doc explaining why we are only reporting half of the network and not all of it. I get it but only because I read the ticket).

Reply:
yeah, should be call `half_network`
ok, will add doc to explain why half